models/interlibrary_loan_requests.rb

```diff
@@ -0,0 +1,54 @@
+class InterlibraryLoanRequests
```

Comment:
All of the public methods in this class should be tested in a /spec/models/interlibrary_loan_requests_spec.rb